src/mono/wasm/templates/templates/browser/browser.0.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk.WebAssembly">
   <PropertyGroup>
-    <TargetFramework>net8.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
could you please elaborate ?

Reply:
https://github.com/dotnet/runtime/blob/77d836c65565c389b0533acbf340635d83ae6ab9/src/mono/wasm/templates/templates/browser/.template.config/template.json#L48-L51

This is what we have in the template config, and it expects to replace `"net7.0"` string. It was incorrectly changed to `net8.0`, but it didn't affect anything because the replacement value was also `net8.0`.